counterpartylib/lib/micropayments/__init__.py

```diff
@@ -0,0 +1,18 @@
+# coding: utf-8
+# Copyright (c) 2016 Fabian Barkhau <fabian.barkhau@gmail.com>
+# License: MIT (see LICENSE file)
+
+
+from .control import make_deposit  # NOQA
```

Comment:
The #NOQA comment is for automated pep8 tests (I fail my builds if they are not pep8 compliant). In thes case I import the function to make it available in the module and pep8 would complain I imported something I didn't use.